edb/server/dbview/dbview.pyx

```diff
@@ -301,6 +302,23 @@ cdef class Database:
             max_batch_size=100,
         )
 
+    cdef _set_extensions(self, extensions):
+        # Update metrics about extension use
+        tname = self.tenant.get_instance_name()
```

Comment:
Did you mean to use `tname` below instead of calling `get_instance_name` in the two loops?

Reply:
Indeed. Thanks for the catch